chef/cookbooks/crowbar/templates/default/crowbar-shutdown-services-before-upgrade.sh.erb

```diff
@@ -48,11 +48,11 @@ fi
 # Note that for HA setup, they should be stopped by pacemaker.
 log "Stopping OpenStack services..."
 
-for i in /etc/init.d/openstack-* \
-         /etc/init.d/apache2 \
-         /etc/init.d/rabbitmq-server \
+for i in /usr/sbin/rcopenstack-* \
+         /usr/sbin/rcapache2 \
+         /usr/sbin/rcrabbitmq-server \
          /etc/init.d/ovs-usurp-config-* \
```

Comment:
BTW, I do not know what this is and why we are stopping it here...

Reply:
this services should no longer exists I believe, reading the neutron recipe (both 4.0 and master) https://github.com/crowbar/crowbar-openstack/blob/stable/4.0/chef/cookbooks/neutron/recipes/common_agent.rb#L131

Reply:
That makes sense, I could not find it anywhere